variables.json

```diff
@@ -174,6 +174,21 @@
                                              "default":true
                                           }
                                        }
+                                    },
+                                    "popupHeight":{
+                                       "title":"Popup Height",
+                                       "type":"string",
+                                       "default":"486px"
+                                    },
+                                    "popupZIndex":{
+                                       "title":"Popup Z Index",
+                                       "type":"integer",
+                                       "default":1
+                                    },
+                                    "popupMarginTop":{
```

Comment:
This has to be something more flexible. Some people might want to add other margins

Reply:
sounds good 👍

Reply:
done!